src/design-guidelines/shadows/shadows-and-depth.md

```diff
@@ -13,27 +14,30 @@ Despite skeuomorphism having already been replaced with more minimalistic UI des
 
 We may look at a subtle visual clue such as an elevated button with a slight shadow underneath it, and semi-consciously perceive it as an interactable element of the user interface. This is because the elevation reminds us of a real physical button. Just by looking at it, we realize that this thing can be pressed, moved, or otherwise interacted with. We may see a toolbar or a modal placed on top of the rest of the elements on the screen, casting a slight shadow on whatever is below it. This helps us understand that the element is higher in the hierarchy, is more important, or demands more attention right now.
 
-
 ## Shadows
+
 When correctly used, shadows can improve usability of UI elements. We can use shadows to:
-- create a sense of depth,
-- improve users' perception of clickable elements,
-- emphasize the hierarchy of sections of the user interface,
-- signal that a component is temporary, by placing it "on top of" a context,
-- clarify the direction of movement of an element, whether it is animated as a result of a user interaction, such as pressing a button, or to indicate transition from one state to another,
-- and even visualize distinct surfaces by clarifying their edges.
+
+-   create a sense of depth,
```

Comment:
🤔 don't know. Did I add it now? I think it was there right? Lint was doing some fixes automatically. Maybe one of them?

Reply:
It was added now… 🤷‍♂️

No biggie.

Reply:
Ah, this was the commit message:

> chore(design-guidelines): lint markdown code in the documentations
>
> Seems to be some change on our lint settings.
> It happened automatically, when I was updating information here.

Just to satisfy my curiosity, I removed this commit, and then ran `npm run lint:fix`, and it did not make any changes to this file. So I'm guessing this happened due to some plugin you have in VSCode. I've actually wanted to include our markdown files in the linting and formatting for a long time, so it's nice in a way. It would be nice to have it controlled by our lint script though, so it doesn't just happen on some people's computers and not on other's. But let's not worry about that right now. Just be aware that you might end up with loads of formatting changes in a markdown file if you change anything in it. But it looks to me like you managed to isolate the formatting changes to its own commit, so that's great! 👍